src/components/BodyComponent.jsx

```diff
@@ -1,94 +1,20 @@
-import React from 'react'
+import React, { useState } from 'react'
 import CardComponent from './CardComponent'
+import data from '../data.json'
+import SearchBar from './SearchBar'
 const BodyComponent = () => {
-    const data = [
-        {
-          img:"" ,
-          name:"Namrata Das",
-          company:"Persistent Systems Limited",
-          location:"Assam",
-        
-        },
-        {
-          img:"" ,
-          name:"Pranav Sonawnane",
-          company:"Onsite electro services pvt Ltd ",
-          location:"Maharashtra",
-        
-        },
-        {
-          img:"" ,
-          name:"Atul",
-          company:"Nagarro Software Pvt. Ltd.",
-          location:"Haryana",
-        
-        },
-        {
-          img:"" ,
-          name:"Gaurav K",
-          company:"",
-          college:"IIT Kanpur",
-          location:"Gadarpur",
-        
-        },
-        {
-          img:"" ,
-          name:"Bhavesh Singh",
-          company:"",
-          college:"Lokmanya Tilak clg of engineering ",
-          location:"Parel",
-        },
-        {
-          img:"" ,
-          name:"Rohan Patil",
-          company:"TCS",
-          location:"Nippani",
-        
-        
-        },
-        {
-          img:"" ,
-          name:"Firdos",
-          company:"Delloite",
-          location:"Madanapalle",
-        
-        
-        },
-        {
-          img:"" ,
-          name:"Dhruv Chaurasia",
-          company:"Saharanpur",
-          location:"Athenahealth",
-        
-        
-        },
-        {
-          img:"" ,
-          name:"Komal",
-          company:"Coforge",
-          location:"Bangalore",
-        
-        
-        },
-        {
-          img:"" ,
-          name:"Suyash Pradhan",
-          company:"Bureau, Inc",
-          location:"Maharashtra",
-        
-        
-        }
-      ]
+   const [searchData, setsearchData] = useState(data)
       
   return (
+    <>
+    <SearchBar setsearchData={setsearchData}/>
     <div className='card-container'>
-   {
-    data.map((items,index)=>{return <CardComponent items={items}/>})
-   }
-  {/* <CardComponent data={data[0]} />
-  <CardComponent data={data[1]}/>
-  <CardComponent data={data[2]}/> */}
-  </div>
+    {
+      searchData.map((items,index)=>{return <CardComponent items={items}/>})
```

Comment:
What if searchData is an empty array? How will you handle it? It will break the application

Reply:
Great observation, will work upon it.